pyproject.toml

```diff
@@ -167,9 +180,10 @@ test = { cmd = "pytest -vsl --cov=skrub --cov-report=xml skrub" }
 lint = ["lint"]
 doc = ["optional", "doc"]
 test = ["test"]
-dev = ["dev", "optional", "doc", "lint", "test"]
+dev = ["dev", "optional", "doc", "lint", "test", "transformers"]
 ci-py309-min-deps = ["py309", "min-dependencies", "test"]
 ci-py309-min-optional-deps = ["py309", "min-dependencies", "min-optional-dependencies", "test"]
+ci-py311-transformers = ["py311", "transformers", "test"]
```

Comment:
OK so it seems that we get closer to a working CI. However, with

```
numpy = ">=1.23.5,<2"
pytorch = { version = "*", channel = "pytorch" }
transformers = "*"
sentence-transformers = "*"
```

We get a old `sentence-transformers` (2.0.0). I'm going to try forcing it to have it to API 3.0 and the latest pytorch.

Reply:
OK I think it works.